scripts/lib/uaa_sync_admin_users.rb

```diff
@@ -176,7 +176,8 @@ def update_admin_users(users)
     to_delete_user_ids.each { |user_id|
       user_info = self.get_user_by_id(user_id)
       if user_info.nil?
-        raise "User #{user_id} not found"
+        get_logger.info("User #{user_id} not found. Skipping.")
+        continue
```

Comment:
In ruby `continue` is spelled `next`:

~~~
$ cat foo.rb
['foo', 'bar', 'baz'].each do |v|
  continue if v == 'bar'
  puts v
end

$ ruby foo.rb
foo
foo.rb:2:in `block in <main>': undefined local variable or method `continue' for main:Object (NameError)
        from foo.rb:1:in `each'
        from foo.rb:1:in `<main>'

$ cat foo_fixed.rb
['foo', 'bar', 'baz'].each do |v|
  next if v == 'bar'
  puts v
end

$ ruby foo_fixed.rb
foo
baz
~~~